Jenkinsfile

```diff
@@ -29,42 +29,60 @@ pipeline {
                 }
             }
         }
+        stage("Build Image & Deploy") {
+            steps {
+                /* can't do that yet
+                checkout scm
+                sh 'sbt universal:packageZipTarball'
+                sh "mv modules/service/target/universal/admin-service-*.tgz ./admin-service.tgz"
+                sh "oc start-build admin-build --from-file=admin-service.tgz --follow"
+                openshiftTag srcStream: 'admin', srcTag: 'latest', destStream: 'admin', destTag: version.substring(1), verbose: 'false'
+                
+                sh "oc project bbp-nexus-dev"
+                sh "oc scale statefulset admin --replicas=0"
+                sleep 2
+                sh "oc scale statefulset admin --replicas=1"
+                sh '''
+                    while true; do
+                        replicas=$(oc get statefulset/admin -o=jsonpath='{.status.readyReplicas}')
```

Comment:
This would not be enough. It will be set to one as soon as the action is triggered.
I think something on the lines of `oc get pods iam-0 --template={{.status}} | grep -c "ready:true"` would be safer